File: fastmq-storage/src/main/java/com/song/fastmq/storage/storage/support/OffsetStorageException.java
package com.song.fastmq.storage.storage.support;

/**
 * @author song
 */
public class OffsetStorageException extends Exception{

    public OffsetStorageException(String message) {
        super(message);
    }

    public OffsetStorageException(Throwable cause) {
        super(cause);
    }

    public OffsetStorageException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: fastmq-storage/src/main/java/com/song/fastmq/storage/storage/support/OffsetStorageException.kt
package com.song.fastmq.storage.storage.support

/**
 * @author song
 */
class OffsetStorageException : Exception {

    constructor(message: String) : super(message) {}

    constructor(cause: Throwable) : super(cause) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}
}
